prow/scripts/cluster-integration/kyma-upgrade-gardener-azure.sh

```diff
@@ -390,10 +396,6 @@ fi
 createTestResources
 
 upgradeKyma
-if [[ "$?" -ne 0 ]]; then
```

Comment:
Why did you remove this?

Reply:
Good point. I forgot to commit one change which makes it clear why this error check can be removed.


Some context:

Previously the stackdriver installation overrides were created only during the upgrade (there was a change in installation overrides, therefore it was not possible to pass the overrides to Kyma 1.15.1, but for current master - and upgrade takes master). So the exit code of the prepare_stackdriver_logging was checked and returned. In case of an error (return 1), the process was exited with 1.

Reply:
As discussed: we will have leave it in case the check installation step throws an error.